### ISSUE TITLE
No link to download files

### ISSUE THREAD

**Issue by Nithyavasudevan** [okfn/ckanext-glasgow#108]
Uploaded a file 'tt' to dataset Architecto laudantium tempore aut
Unable to find a link to download it.
![image](https://cloud.githubusercontent.com/assets/7315913/3904070/47090546-22d4-11e4-8b76-36cce5225ece.png)
![image](https://cloud.githubusercontent.com/assets/7315913/3904077/5e304c70-22d4-11e4-8aea-0ad7a6e651f8.png)
![image](https://cloud.githubusercontent.com/assets/7315913/3904084/6d681092-22d4-11e4-9f83-956b27b48d30.png)

**Comment by Nithyavasudevan** [comment]
Please note that clicking filename also takes me to the audit page
![image](https://cloud.githubusercontent.com/assets/7315913/3905542/0641bd36-22e9-11e4-9835-7bb5b17605f0.png)

**Comment by amercader** [comment]
When you upload a new file, the link that appears is created under "Pending Resources". This means that the file creation request is being processed on the Platform. That's why we link to the request status check.

Once the file is approved in the platform _and_ created in CKAN by the changelog harvester, the file will appear under the "Data and Resources" section.

Right now this is not happening because
1. Files seem to get stuck for a long time with an `InProgress` status on the platform
2. The changelog harvester can not create the files in CKAN because the relevant query to get the file metadata from the platform was not working and we need to update it

So in summary, once the whole file creation workflow works as expected, the file links will appear in the above "Data and Resources" section.

**Comment by Nithyavasudevan** [comment]
I am guessing this will be in I5?

**Comment by amercader** [comment]
Yes
